editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx

```diff
@@ -39,8 +60,16 @@ export const FaviconForm: React.FC = () => {
       input={
         <InputRow>
           <InputRowLabel>Favicon:</InputRowLabel>
-          <InputRowItem width={50}>
-            <PublicFileUploadButton />
+          <InputRowItem width={formik.values.favicon ? 90 : 50}>
```

Comment:
Resizing this on upload looks better to me than keeping it wide the whole time - very open to opinions here if this jump in not consistent / a bit weird @ianjon3s